lib/message/fpErrorNotifierMessageHelper.php

```diff
@@ -64,7 +64,9 @@ public function formatServer()
   {
     return array(
       'server' => $this->dump($_SERVER),
-      'session' => $this->dump(isset($_SESSION) ? $_SESSION : null));
+      'session' => $this->dump(isset($_SESSION) ? $_SESSION : null),
+      'post' => $this->dump($_POST)
```

Comment:
should this be checked with an isset like the session stuff on the line above? (ie what if there is no postdata)